src/mots/config.py

```diff
@@ -34,7 +33,7 @@ class ValidationError(TypeError):
 class FileConfig:
     """Loader and writer for filesystem based configuration."""
 
-    def __init__(self, path: Path = DEFAULT_CONFIG_FILEPATH):
+    def __init__(self, path: Path = Path(settings.DEFAULT_CONFIG_FILEPATH)):
```

Comment:
This should already be `Path`, no need to wrap it.

Reply:
Indeed redundant, forgot to update after changing the type.